conan/api/subapi/config.py

```diff
@@ -10,13 +10,14 @@ def home(self):
         return self.conan_api.cache_folder
 
     def install(self, path_or_url, verify_ssl, config_type=None, args=None,
-                source_folder=None, target_folder=None):
+                source_folder=None, target_folder=None, strategy="replace"):
```

Comment:
I think the approach is not that much as cli args, but as information encoded in files.